pkg/server/k8s/converters_test.go

```diff
@@ -103,6 +103,25 @@ func TestPodSpecVolumesToK8sVolumes(t *testing.T) {
 	}
 }
 
+func TestPodSpecEmptyDirVolumeToK8s(t *testing.T) {
+	vols := []*spec.Volume{
+		{Name: "Vol-Test", EmptyDir: true},
+	}
+	k8sVols := podSpecVolumesToK8sVolumes(vols)
+
+	for idx, vol := range vols {
```

Comment:
Maybe remove this for?

Reply:
In fact, the function is slice based (the slices are not used for test cases) so I guess is easier to use a loop for testing.

Reply:
Make sense!